spec/features/cart/show_spec.rb

```diff
@@ -27,24 +35,23 @@
 
       visit cart_path
 
-      [@i1,@i2].each do |item|
```

Comment:
Yeaaa, I realize in hindsight that iterating over items could possibly make it more difficult to test in these block. Thanks for the change, I'll definitely rethink if I should do this kind of iterating next time.

Reply:
It was a fancy idea though!